Source/WebKit/Shared/cf/CFTypes.serialization.in

```diff
@@ -40,6 +40,9 @@
 [WebKitPlatform, CustomHeader, AdditionalEncoder=StreamConnectionEncoder, ToCFMethod=result->createCFURL()] CFURLRef wrapped by WebKit::CoreIPCCFURL {
 }
 
+[WebKitPlatform, CustomHeader, AdditionalEncoder=StreamConnectionEncoder, ToCFMethod=result->createSecCertificate()] SecCertificateRef wrapped by WebKit::CoreIPCSecCertificate {
```

Comment:
I don't know what `AdditionalEncoder=StreamConnectionEncoder` does, but all the other CFTypes had this so I kept it for `SecCertificateRef`.

Reply:
It's needed if the object is serialized to GPUP.

Reply:
Yup, doesn't compile if I remove that, so I'll leave it in.